client/components/list-end/docs/example.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import ListEnd from 'components/list-end';
+
+export default function ListEndExample() {
+	return <ListEnd />;
+}
+ListEndExample.displayName = 'ListEndExample';
```

Comment:
The issue is that for minified builds function names are mangled. That would lead to the following in devdocs on wpcalypso:

![mangled](https://user-images.githubusercontent.com/841763/28824946-a85519ec-76c4-11e7-94ae-a44636903eef.png)

I've just tested this on a local docker build.